docs/modules/ROOT/pages/how-tos/configuring/prefetching-dependencies.adoc

```diff
@@ -298,6 +302,58 @@ spec:
 * From the {ProductName} *Applications* view, go to *Activity > Latest commits*.
 //include::../partials/con_hermetic_verification.adoc[]
 
+== [[yarn]]Enabling prefetch builds for `yarn`
+
+Supported versions: 1.x and 3.x. Cachi2 automatically detects the version of `yarn` and fetches any dependencies you declare in your `package.json` and `yarn.lock` project files.
+
+.Prerequisites
+
+* You have an up-to-date `yarn.lock` file in your source repository. To make sure that you have the latest `yarn.lock` file, or to create a lockfile, run the `yarn install` command. If `yarn.lock` is not up-to-date, Cachi2 will not fetch the dependencies.
+
+.Procedure
+
+To prefetch dependencies for a component build, complete the following steps:
+
+. Go to the `.tekton` directory and find the `.yaml` files related to the `*pull request*` and `*push*` processes.
+. Configure the hermetic pipeline by adding the following parameters in both `.yaml` files:
+
++
+[source,yaml]
+----
+spec:
+  params:
+    - ...
+    - name: prefetch-input
+      value: '{"type": "yarn", "path": "."}'
+----
```

Comment:
Is this basically the same across all package managers? Would it be possible to abstract the general form to a common location and then just have sections on the individuality of the different managers?

Reply:
Yes, there is *a lot* of repetition in the docs. Each package manager has the same paragraph:
```
Verification
From the Konflux Applications view, go to Activity > Pipeline runs.

Go to the pipeline run with Build in the Type column and confirm that the prefetch-dependencies stage displays a green checkmark. This indicates that the build process successfully fetched all dependencies.

From the Konflux Applications view, go to Activity > Latest commits.
```
So I would propose a follow-up to clean this up and overall make it more concise. Unless you favor doing it here.
But I can delete the `Procedure` section since it is duplicated for all package managers as well.

Reply:
A follow up is fine if you don't want to do it righ tnow. You are being consistent.